vkubelet/pod.go

```diff
@@ -204,7 +205,7 @@ func (s *Server) updatePodStatus(ctx context.Context, pod *corev1.Pod) error {
 	ctx = addPodAttributes(ctx, span, pod)
 
 	status, err := s.provider.GetPodStatus(ctx, pod.Namespace, pod.Name)
-	if err != nil {
+	if err != nil && !strongerrors.IsNotFound(err) {
```

Comment:
Can you add to the docs somewhere that `GetPod` / `GetPodStatus` should return a `NotFoundError`?

Reply:
See #635